components/brave_ads/browser/ads_notification_handler.cc

```diff
@@ -67,55 +64,22 @@ void AdsNotificationHandler::OnClick(
     const base::Optional<base::string16>& reply,
     base::OnceClosure completed_closure) {
```

Comment:
what happens to the completed callback?

Reply:
This is legacy code and as above will be removed when native ad notifications are removed